lib/rex/post/meterpreter/ui/console/command_dispatcher/core.rb

```diff
@@ -856,18 +871,18 @@ def cmd_migrate(*args)
 
     # Do this thang.
     if client.platform =~ /linux/
-      client.core.migrate(pid, writable_dir)
+      client.core.migrate(pid, writable_dir, opts)
     else
-      client.core.migrate(pid)
+      client.core.migrate(pid, opts: opts)
```

Comment:
I thought the use of named parameters here was required given that opts is
the third parameter, not the second?